### PR TITLE
add highlight for book and translation

### PR DESCRIPTION
**Pull request by YusongWang**
## Motivation

add highlight for book

### Have you read the [Contributing Guidelines on pull requests](https://github.com/move-language/move/blob/main/CONTRIBUTING.md#developer-workflow)?

Yes

## Test Plan

No code change

### CODE REVIEW

**Comment by YusongWang** [comment]
@wrwg @jolestar can somebody review this change and let me merge?

**Comment by wrwg** [comment]
Adding some Chinese speakers

**Comment by jolestar** [comment]
@geometryolife help to review.

File: language/documentation/book/translations/move-book-zh/src/abort-and-assert.md [hunk 1]
```diff
@@ -22,26 +22,25 @@ abort 42
...
-`abort` 表达式会中止执行当前函数并恢复当前事务对全局状态所做的所有更改。Move语言没有“捉捕”或者额外处理`abort`的机制。
+`abort` 表达式会中止执行当前函数并恢复当前事务对全局状态所做的所有更改。Move 语言没有“捉捕”或者额外处理`abort`的机制。
 
 Luckily, in Move transactions are all or nothing, meaning any changes to global storage are made all
 at once only if the transaction succeeds. Because of this transactional commitment of changes, after
 an abort there is no need to worry about backing out changes. While this approach is lacking in
 flexibility, it is incredibly simple and predictable.
 
-幸运的是，在Move里事务的计算要么完全执行要么完全不执行。这意味着只有在事务成功时，任何对全局存储状态的改变才会被一并执行。
+幸运的是，在 Move 里事务的计算要么完全执行要么完全不执行。这意味着只有在事务成功时，任何对全局存储状态的改变才会被一并执行。
```
Comment (geometryolife) [inline language/documentation/book/translations/move-book-zh/src/abort-and-assert.md:32]:
```suggestion
幸运的是，在 Move 里交易的计算要么完全执行要么完全不执行。这意味着只有在交易成功时，任何对全局存储状态的改变才会被一并执行。
```

**Review by geometryolife — COMMENTED** [review]
Already reviewed.

**Comment by vgao1996** [comment]
The changes (both English & Chinese) look good to me.

Also happy to see the syntax highlighter being added. It looks like the highlighter is generated by some tool. Would you mind sharing what that tool is? Wonder if we could also include the original highlighter rules in the repo.

**Comment by YusongWang** [comment]
> The changes (both English & Chinese) look good to me.
> 
> Also happy to see the syntax highlighter being added. It looks like the highlighter is generated by some tool. Would you mind sharing what that tool is? Wonder if we could also include the original highlighter rules in the repo.

The repo is here [highlightjs-move](https://github.com/movefuns/highlightjs-move), i'am not put any readme yet. this repo is a plug-in for highlight.js and  only have one file call move.js. the rules here

**Comment by YusongWang** [comment]
@vgao1996 @geometryolife Hey guys, can somebody approving this?

**Review by runtian-zhou — APPROVED** [review]

**Review by vgao1996 — APPROVED** [review]

**Comment by YusongWang** [comment]
@wrwg Hey man, can you trigger wf again?

**Pull request closed without merging**